app/views/todo_tasks/_form_edit.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= form_with(model: todo_task) do |form| %>
+  <% if todo_task.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(todo_task.errors.count, "error") %> prevented this Task from being updated:</h2>
+
+      <ul>
+        <% todo_task.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <%= form.label :description, style: "display: block" %>
+    <%= form.text_field :description %>
+  </div>
+
+  <div>
+    <%= form.label :reminder, style: "display: block" %>
+    <%= form.datetime_field :reminder %>
+  </div>
+
+  <div>
+    <%= form.label :dueby, style: "display: block" %>
+    <%= form.datetime_field :dueby %>
+  </div>
+
+  <div>
+    <%= form.label :completed, style: "display: block" %>
+    <%= form.check_box :completed %>
```

Comment:
Include second form for edit so user can update completion status. 

Q: can this be consolidated into one form (_form.html.erb & _form_edit.html.erb) and conditionally render completed field based on parameter?